<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelODF</class>
 <widget class="QScrollArea" name="PanelODF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScrollArea</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="widgetResizable">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="scrollAreaWidgetContents">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>374</width>
     <height>449</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutToolbar">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolBar" name="toolbar">
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="floatable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="6" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Inversion</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSpinBox" name="spinBoxSkip">
          <property name="maximumSize">
           <size>
            <width>45</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <widget class="QWidget" name="widgetMaskControl" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>125</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Mask threshold</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditMaskThreshold">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="comboBoxColorCode">
        <item>
         <property name="text">
          <string>Directional</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Magnitude</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="labelSolidColor">
        <property name="text">
         <string>Color</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Skip</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFileName">
        <property name="text">
         <string>File name</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditFileName">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxMask">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="lineEditScale">
          <property name="maximumSize">
           <size>
            <width>45</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="labelColorMin">
        <property name="text">
         <string>Min</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Mask</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="comboBoxInversion">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Invert X</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Invert Y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Invert Z</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Scale</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSlider" name="sliderColorMin">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditColorMin">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="labelColorMax">
        <property name="text">
         <string>Max</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QSlider" name="sliderColorMax">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditColorMax">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>240</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliderColorMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PanelODF</receiver>
   <slot>OnSliderColorThreshold(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderColorMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PanelODF</receiver>
   <slot>OnSliderColorThreshold(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditColorMin</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PanelODF</receiver>
   <slot>OnLineEditColorThreshold(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditColorMax</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PanelODF</receiver>
   <slot>OnLineEditColorThreshold(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>678</x>
     <y>506</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnSliderColorThreshold(int)</slot>
  <slot>OnLineEditColorThreshold(QString)</slot>
 </slots>
</ui>
